smart-contract-extensions/test/unlock-examples/hooks/codeRequiredHook.js

```diff
@@ -0,0 +1,110 @@
+const { constants, protocols } = require('hardlydifficult-ethereum-contracts')
+const erc1820 = require('erc1820')
+const { reverts } = require('truffle-assertions')
+
+const CodeRequiredHook = artifacts.require('CodeRequiredHook.sol')
+
+contract('CodeRequiredHook', accounts => {
+  const lockOwner = accounts[1]
```

Comment:
Minor, but could we maybe use lockCreator here moving forward? 
We'll be doing a fairly large migration away from using "owner" fairly soon. 
"lockCreato"r is pretty descriptive, and may help with the transition away from using Ownable, at least cognitively.